### ISSUE TITLE
Changes related to Capacity Reservation resource to display maximum fault domain count supported

### ISSUE THREAD

**Issue by hari-bodicherla** [Azure/azure-cli#23414]
**Related command**

**Is your feature request related to a problem? Please describe.**
we currently support VMSS deployments against Capacity Reservation with FD count depending up on if the VM Size supports tenant spanning or not . So the max fd count will be 1 if tenant spanning doesnt exist for the vm size in the CR and if yes, max fd count will be 3. This is ok for now, but in feature, we need to restrict the fd count to 2 for some speciality skus due to vm sizes being special and capacity constraints issues in the regions. Due to this, we need a property which can be managed and populated from platform side and returned to customer. This will be readonly property for customer. So Customer or client tools like portal should know the max fd count that we support when using the specific capacity reservation. The proposed changes in this PR ensure we return the max fd count supported in the Get CR response which can be used by client tools and customers to decide if they can go ahead with the resource deployments with the CR.

**Describe the solution you'd like**
we are adding a readonly property which can be managed and populated from platform side and returned to customer. So Customer or client tools like portal should know the max fd count that we support when using the specific capacity reservation. The proposed changes in this PR ensure we return the max fd count supported in the Get CR response which can be used by client tools and customers to decide if they can go ahead with the resource deployments with the CR.

**Describe alternatives you've considered**
Customer can do trial and error on recieving the client or server errors to change the desired FD and vm size when using the Capacity Reservation group.

**Additional context**
I am not able to find any cmdlet for adding the param in the Get Capacity Reservation response. Please let me know if this review is not needed for the platform generated properties in the response.

**Comment by yonzhan** [comment]
Compute

**Comment by zhoxing-ms** [comment]
@hari-bodicherla Could you please provide the Swagger PR link, the design of CLI commands and parameters, the minimum api-version required, and the expected ETA?

In addition, it is recommended that you use the [[Microsoft Internal / Service Team] Feature Request](https://github.com/Azure/azure-cli/issues/new?assignees=&labels=&template=Service_team_request.yaml) to create internal feature requests, so we can better understand the information of this demand, such as this feature request #23394 22275

**Comment by hari-bodicherla** [comment]
> @hari-bodicherla Could you please provide the Swagger PR link, the design of CLI commands and parameters, the minimum api-version required, and the expected ETA?
> 
> In addition, it is recommended that you use the [[Microsoft Internal / Service Team] Feature Request](https://github.com/Azure/azure-cli/issues/new?assignees=&labels=&template=Service_team_request.yaml) to create internal feature requests, so we can better understand the information of this demand, like this feature request #22275 ![Screenshot 2022-08-03 104241](https://user-images.githubusercontent.com/16612065/182514084-158aaf02-af26-4fb2-90be-6b3fdbc3a1b0.png)


@zhoxing-ms Here are the Details

Swagger link: https://github.com/Azure/azure-rest-api-specs/pull/20076 

Design:
There wont be any change in the existing cmdlets , since this is a readonly property that is set by the Platform and return it in the response to customer for az capacity reservation show cmdlet.
The new readonly property is maximumFaultDomainCountSupported

Expected ETA:
Earliest Date for CLI release.

Minimum API version required:
2022-08-01

**Comment by zhoxing-ms** [comment]
@hari-bodicherla Currently, the latest version of Python SDK does not contain the folder of api-version `2022-08-01`. And I see that the `readme.py` is not modified in this PR https://github.com/Azure/azure-rest-api-specs/pull/20076/files , so it can't release a new SDK package version for this api-version folder.
Please note that since CLI is dependent on the Python SDK, so please contact the python SDK team @msyyc to release a new SDK package version which containing the `2022-08-01` version folder, thanks

**Comment by hari-bodicherla** [comment]
Hi @Theodore ***@***.***>,

Can you please let us know if you are going to trigger the request for Python SDK once the swagger PR is merged at https://github.com/Azure/azure-rest-api-specs/pull/20076 . Also Can you please answer the question from @Xing ***@***.***> on why the readme.py has not modified in the 2022-08-01 branch swagger files.

Thanks,
Hari B

**Comment by hari-bodicherla** [comment]
@Hari Krishna ***@***.***>, yes I will be submitting a request for all SDKs to be generated/release once swgger is merged.

@Xing ***@***.***> what are you expecting in the readme.py file? We usually only add new version tags in the readme.md file and submit a openAPI request to have Go, Java, JS, Python sdk generated. Having the readme.py updated is new to me. What needs to be updated?

From: Hari Krishna Bodicharla ***@***.***>
Sent: Tuesday, August 9, 2022 11:49 AM
To: Azure/azure-cli ***@***.***>; Azure/azure-cli ***@***.***>; Theodore Chang ***@***.***>; Xing Zhou ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)


Hi @Theodore ***@***.***>,

Can you please let us know if you are going to trigger the request for Python SDK once the swagger PR is merged at https://github.com/Azure/azure-rest-api-specs/pull/20076<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F20076&data=05%7C01%7CTheodore.Chang%40microsoft.com%7C3b0a0d9fbc2049bfb36608da7a1e9fc0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637956569194148560%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=HAJH5dvtdw050NX8wM3azO2UFBGqfb9Glow%2Bdald3j8%3D&reserved=0> . Also Can you please answer the question from @Xing ***@***.***> on why the readme.py has not modified in the 2022-08-01 branch swagger files.

Thanks,
Hari B

**Comment by hari-bodicherla** [comment]
Hi @Theodore ***@***.***>,

For how to modify reamde.py to release a new api-version folder for a new Python SDK package, please ask Python SDK team @Yuchao ***@***.***> for help~

Thanks,
Xing

From: Theodore ***@***.***>
Sent: 2022年8月10日 5:10
To: Hari Krishna ***@***.***>; ***@***.***>; ***@***.***>; Xing ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

@Hari Krishna ***@***.***>, yes I will be submitting a request for all SDKs to be generated/release once swgger is merged.

@Xing ***@***.***> what are you expecting in the readme.py file? We usually only add new version tags in the readme.md file and submit a openAPI request to have Go, Java, JS, Python sdk generated. Having the readme.py updated is new to me. What needs to be updated?

From: Hari Krishna Bodicharla ***@***.***>
Sent: Tuesday, August 9, 2022 11:49 AM
To: Azure/azure-cli ***@***.***>; Azure/azure-cli ***@***.***>; Theodore Chang ***@***.***>; Xing Zhou ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)


Hi @Theodore ***@***.***>,

Can you please let us know if you are going to trigger the request for Python SDK once the swagger PR is merged at https://github.com/Azure/azure-rest-api-specs/pull/20076<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F20076&data=05%7C01%7CZhou.Xing%40microsoft.com%7Ca9c7cb495f23428d571708da7a4ba0f4%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637956762505023164%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2B35gsDsknoa8jr6zZ4cQNRkXOwIsIEgYM494FdlwvRY%3D&reserved=0> . Also Can you please answer the question from @Xing ***@***.***> on why the readme.py has not modified in the 2022-08-01 branch swagger files.

Thanks,
Hari B

**Comment by hari-bodicherla** [comment]
Hi, @Theodore ***@***.***>

Yes, please add it with reference of other tags.

Yuchao
From: Theodore Chang ***@***.***>
Sent: Wednesday, August 10, 2022 11:59 PM
To: Xing Zhou ***@***.***>; Hari Krishna Bodicharla ***@***.***>; Azure/azure-cli ***@***.***>; Azure/azure-cli ***@***.***>; Yuchao Yan ***@***.***>
Cc: Yong Zhang ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

@Yuchao ***@***.***>,

In the past, we never updated the readme.python.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fblob%2Ffeature%2Fcplat-2022-08-01%2Fspecification%2Fcompute%2Fresource-manager%2Freadme.python.md&data=05%7C01%7Cyuchaoyan%40microsoft.com%7Ca9f7f75289c94bbb78a408da7ae93ff9%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637957439500952086%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=eEiOnatnM8w3j88aeBjgZDdgl7MMH%2B1IXSaSqaFDryQ%3D&reserved=0> file, but I see there are recent package tags. Do we just need to add tag for package-2022-08-01-only here?



From: Xing Zhou ***@***.******@***.***>>
Sent: Tuesday, August 9, 2022 11:49 PM
To: Theodore Chang ***@***.******@***.***>>; Hari Krishna Bodicharla ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>
Cc: Yong Zhang ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

Hi @Theodore ***@***.***>,

For how to modify reamde.py to release a new api-version folder for a new Python SDK package, please ask Python SDK team @Yuchao ***@***.***> for help~

Thanks,
Xing

From: Theodore ***@***.***>
Sent: 2022年8月10日 5:10
To: Hari Krishna ***@***.***>; ***@***.***>; ***@***.***>; Xing ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

@Hari Krishna ***@***.***>, yes I will be submitting a request for all SDKs to be generated/release once swgger is merged.

@Xing ***@***.***> what are you expecting in the readme.py file? We usually only add new version tags in the readme.md file and submit a openAPI request to have Go, Java, JS, Python sdk generated. Having the readme.py updated is new to me. What needs to be updated?

From: Hari Krishna Bodicharla ***@***.******@***.***>>
Sent: Tuesday, August 9, 2022 11:49 AM
To: Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Theodore Chang ***@***.******@***.***>>; Xing Zhou ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)


Hi @Theodore ***@***.***>,

Can you please let us know if you are going to trigger the request for Python SDK once the swagger PR is merged at https://github.com/Azure/azure-rest-api-specs/pull/20076<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F20076&data=05%7C01%7Cyuchaoyan%40microsoft.com%7Ca9f7f75289c94bbb78a408da7ae93ff9%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637957439500952086%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=glCNxeUI5WglvKTxGzDEJPOxjyxCc%2FPEAZBAMJWqG3U%3D&reserved=0> . Also Can you please answer the question from @Xing ***@***.***> on why the readme.py has not modified in the 2022-08-01 branch swagger files.

Thanks,
Hari B

**Comment by hari-bodicherla** [comment]
Hello @Theodore ***@***.***>,

Since this has blocked our developing process, may I ask when will the SDK with api-version 2022-08-01 be released?


Thanks,
Yan

发件人: Xing Zhou ***@***.***>
发送时间: 2022年8月18日 9:40
收件人: Yan Zhu ***@***.***>
主题: FW: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)


FYI

From: Yuchao ***@***.***>
Sent: 11 August 2022 10:41
To: Theodore ***@***.***>; Xing ***@***.***>; Hari Krishna ***@***.***>; ***@***.***>; ***@***.***>
Cc: Yong ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

Hi, @Theodore ***@***.***>

Yes, please add it with reference of other tags.

Yuchao
From: Theodore Chang ***@***.******@***.***>>
Sent: Wednesday, August 10, 2022 11:59 PM
To: Xing Zhou ***@***.******@***.***>>; Hari Krishna Bodicharla ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>
Cc: Yong Zhang ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

@Yuchao ***@***.***>,

In the past, we never updated the readme.python.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fblob%2Ffeature%2Fcplat-2022-08-01%2Fspecification%2Fcompute%2Fresource-manager%2Freadme.python.md&data=05%7C01%7Czhuyan%40microsoft.com%7Ca3597f6aedf04353cd3808da80ba9331%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637963836080123827%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lejrYmqtUQJg4JEQteS6dEk1rF7XRYPuzTfE0QnHK78%3D&reserved=0> file, but I see there are recent package tags. Do we just need to add tag for package-2022-08-01-only here?



From: Xing Zhou ***@***.******@***.***>>
Sent: Tuesday, August 9, 2022 11:49 PM
To: Theodore Chang ***@***.******@***.***>>; Hari Krishna Bodicharla ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>
Cc: Yong Zhang ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

Hi @Theodore ***@***.***>,

For how to modify reamde.py to release a new api-version folder for a new Python SDK package, please ask Python SDK team @Yuchao ***@***.***> for help~

Thanks,
Xing

From: Theodore ***@***.***>
Sent: 2022年8月10日 5:10
To: Hari Krishna ***@***.***>; ***@***.***>; ***@***.***>; Xing ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

@Hari Krishna ***@***.***>, yes I will be submitting a request for all SDKs to be generated/release once swgger is merged.

@Xing ***@***.***> what are you expecting in the readme.py file? We usually only add new version tags in the readme.md file and submit a openAPI request to have Go, Java, JS, Python sdk generated. Having the readme.py updated is new to me. What needs to be updated?

From: Hari Krishna Bodicharla ***@***.******@***.***>>
Sent: Tuesday, August 9, 2022 11:49 AM
To: Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Theodore Chang ***@***.******@***.***>>; Xing Zhou ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)


Hi @Theodore ***@***.***>,

Can you please let us know if you are going to trigger the request for Python SDK once the swagger PR is merged at https://github.com/Azure/azure-rest-api-specs/pull/20076<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F20076&data=05%7C01%7Czhuyan%40microsoft.com%7Ca3597f6aedf04353cd3808da80ba9331%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637963836080123827%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2Fnaj4853DE97%2FOj3udUTNpqA1maPFPdktmd3heFq32c%3D&reserved=0> . Also Can you please answer the question from @Xing ***@***.***> on why the readme.py has not modified in the 2022-08-01 branch swagger files.

Thanks,
Hari B

**Comment by hari-bodicherla** [comment]
Hi @Theodore ***@***.***>,

May I ask when will the SDK with api-version 2022-08-01 be released?


Thanks,
Yan
________________________________
发件人: Yan Zhu
发送时间: 2022年8月18日 11:39
收件人: Xing Zhou ***@***.***>; Hari Krishna Bodicharla ***@***.***>; ***@***.*** ***@***.***>; ***@***.*** ***@***.***>; Yuchao Yan ***@***.***>; Theodore Chang ***@***.***>
抄送: Yong Zhang ***@***.***>
主题: 回复: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)


Hello @Theodore ***@***.***>,



Since this has blocked our developing process, may I ask when will the SDK with api-version 2022-08-01 be released?





Thanks,

Yan



发件人: Xing Zhou ***@***.***>
发送时间: 2022年8月18日 9:40
收件人: Yan Zhu ***@***.***>
主题: FW: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)





FYI



From: Yuchao ***@***.***>
Sent: 11 August 2022 10:41
To: Theodore ***@***.***>; Xing ***@***.***>; Hari Krishna ***@***.***>; ***@***.***>; ***@***.***>
Cc: Yong ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)



Hi, @Theodore ***@***.***>

Yes, please add it with reference of other tags.



Yuchao

From: Theodore Chang ***@***.******@***.***>>
Sent: Wednesday, August 10, 2022 11:59 PM
To: Xing Zhou ***@***.******@***.***>>; Hari Krishna Bodicharla ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>
Cc: Yong Zhang ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)



@Yuchao ***@***.***>,



In the past, we never updated the readme.python.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fblob%2Ffeature%2Fcplat-2022-08-01%2Fspecification%2Fcompute%2Fresource-manager%2Freadme.python.md&data=05%7C01%7Czhuyan%40microsoft.com%7Ca3597f6aedf04353cd3808da80ba9331%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637963836080123827%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lejrYmqtUQJg4JEQteS6dEk1rF7XRYPuzTfE0QnHK78%3D&reserved=0> file, but I see there are recent package tags. Do we just need to add tag for package-2022-08-01-only here?







From: Xing Zhou ***@***.******@***.***>>
Sent: Tuesday, August 9, 2022 11:49 PM
To: Theodore Chang ***@***.******@***.***>>; Hari Krishna Bodicharla ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>
Cc: Yong Zhang ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)



Hi @Theodore ***@***.***>,



For how to modify reamde.py to release a new api-version folder for a new Python SDK package, please ask Python SDK team @Yuchao ***@***.***> for help~



Thanks,

Xing



From: Theodore ***@***.***>
Sent: 2022年8月10日 5:10
To: Hari Krishna ***@***.***>; ***@***.***>; ***@***.***>; Xing ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)



@Hari Krishna ***@***.***>, yes I will be submitting a request for all SDKs to be generated/release once swgger is merged.



@Xing ***@***.***> what are you expecting in the readme.py file? We usually only add new version tags in the readme.md file and submit a openAPI request to have Go, Java, JS, Python sdk generated. Having the readme.py updated is new to me. What needs to be updated?



From: Hari Krishna Bodicharla ***@***.******@***.***>>
Sent: Tuesday, August 9, 2022 11:49 AM
To: Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Theodore Chang ***@***.******@***.***>>; Xing Zhou ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)





Hi @Theodore ***@***.***>,



Can you please let us know if you are going to trigger the request for Python SDK once the swagger PR is merged at https://github.com/Azure/azure-rest-api-specs/pull/20076<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F20076&data=05%7C01%7Czhuyan%40microsoft.com%7Ca3597f6aedf04353cd3808da80ba9331%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637963836080123827%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2Fnaj4853DE97%2FOj3udUTNpqA1maPFPdktmd3heFq32c%3D&reserved=0> . Also Can you please answer the question from @Xing ***@***.***> on why the readme.py has not modified in the 2022-08-01 branch swagger files.



Thanks,

Hari B

**Comment by hari-bodicherla** [comment]
Hi,

If you want a public Python SDK, please follow azure-rest-api-specs/rules-for-release-request.md at main * Azure/azure-rest-api-specs (github.com)<https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/release-request/rules-for-release-request.md> to make a release issue after swagger PR is merged into main branch.

Yuchao
From: Xing Zhou ***@***.***>
Sent: Wednesday, August 10, 2022 11:49 AM
To: Theodore Chang ***@***.***>; Hari Krishna Bodicharla ***@***.***>; Azure/azure-cli ***@***.***>; Azure/azure-cli ***@***.***>; Yuchao Yan ***@***.***>
Cc: Yong Zhang ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

Hi @Theodore ***@***.***>,

For how to modify reamde.py to release a new api-version folder for a new Python SDK package, please ask Python SDK team @Yuchao ***@***.***> for help~

Thanks,
Xing

From: Theodore ***@***.***>
Sent: 2022年8月10日 5:10
To: Hari Krishna ***@***.***>; ***@***.***>; ***@***.***>; Xing ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

@Hari Krishna ***@***.***>, yes I will be submitting a request for all SDKs to be generated/release once swgger is merged.

@Xing ***@***.***> what are you expecting in the readme.py file? We usually only add new version tags in the readme.md file and submit a openAPI request to have Go, Java, JS, Python sdk generated. Having the readme.py updated is new to me. What needs to be updated?

From: Hari Krishna Bodicharla ***@***.******@***.***>>
Sent: Tuesday, August 9, 2022 11:49 AM
To: Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Theodore Chang ***@***.******@***.***>>; Xing Zhou ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)


Hi @Theodore ***@***.***>,

Can you please let us know if you are going to trigger the request for Python SDK once the swagger PR is merged at https://github.com/Azure/azure-rest-api-specs/pull/20076<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F20076&data=05%7C01%7Cyuchaoyan%40microsoft.com%7Cbf442878daaf4f6dce4608da7a833d73%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637957001340819857%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Y2SwnSd0lUrHYmjxvVUJcIDtOpCu4%2FYpxOqBHImUkqo%3D&reserved=0> . Also Can you please answer the question from @Xing ***@***.***> on why the readme.py has not modified in the 2022-08-01 branch swagger files.

Thanks,
Hari B

**Comment by hari-bodicherla** [comment]
@Yuchao ***@***.***>,

In the past, we never updated the readme.python.md<https://github.com/Azure/azure-rest-api-specs/blob/feature/cplat-2022-08-01/specification/compute/resource-manager/readme.python.md> file, but I see there are recent package tags. Do we just need to add tag for package-2022-08-01-only here?



From: Xing Zhou ***@***.***>
Sent: Tuesday, August 9, 2022 11:49 PM
To: Theodore Chang ***@***.***>; Hari Krishna Bodicharla ***@***.***>; Azure/azure-cli ***@***.***>; Azure/azure-cli ***@***.***>; Yuchao Yan ***@***.***>
Cc: Yong Zhang ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

Hi @Theodore ***@***.***>,

For how to modify reamde.py to release a new api-version folder for a new Python SDK package, please ask Python SDK team @Yuchao ***@***.***> for help~

Thanks,
Xing

From: Theodore ***@***.***>
Sent: 2022年8月10日 5:10
To: Hari Krishna ***@***.***>; ***@***.***>; ***@***.***>; Xing ***@***.***>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)

@Hari Krishna ***@***.***>, yes I will be submitting a request for all SDKs to be generated/release once swgger is merged.

@Xing ***@***.***> what are you expecting in the readme.py file? We usually only add new version tags in the readme.md file and submit a openAPI request to have Go, Java, JS, Python sdk generated. Having the readme.py updated is new to me. What needs to be updated?

From: Hari Krishna Bodicharla ***@***.******@***.***>>
Sent: Tuesday, August 9, 2022 11:49 AM
To: Azure/azure-cli ***@***.******@***.***>>; Azure/azure-cli ***@***.******@***.***>>; Theodore Chang ***@***.******@***.***>>; Xing Zhou ***@***.******@***.***>>
Subject: RE: [Azure/azure-cli] Changes related to Capacity Reservation resource to display maximum fault domain count supported (Issue #23414)


Hi @Theodore ***@***.***>,

Can you please let us know if you are going to trigger the request for Python SDK once the swagger PR is merged at https://github.com/Azure/azure-rest-api-specs/pull/20076<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F20076&data=05%7C01%7CTheodore.Chang%40microsoft.com%7Cbf442878daaf4f6dce4608da7a833d73%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637957001362926960%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Y7ynCXHrkesYNxwe%2FvqYrRDpeopX%2BUvmld5GPvQ58%2FE%3D&reserved=0> . Also Can you please answer the question from @Xing ***@***.***> on why the readme.py has not modified in the 2022-08-01 branch swagger files.

Thanks,
Hari B

**Comment by hari-bodicherla** [comment]
Hi @Yishi ***@***.***>, @Ritesh ***@***.***>

Python will catch 3/24 (3/17) release train this time. But Mgmt SDK follow https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/761/Management-SDK-Release-Cycle to release monthly now. So in the future, please make release request earlier and give us more time  to arrange the release.

Add all members of @***@***.***> for awareness about new SDK release policy.

Thanks,
Yuchao
From: Yishi Wang ***@***.***>
Sent: Monday, March 13, 2023 11:18 AM
To: Josephine Qiu ***@***.***>; Ritesh Chitlangia ***@***.***>; Jack Richins ***@***.***>; Nicholas Kondamudi ***@***.***>; Kyle Kim ***@***.***>; Yuchao Yan ***@***.***>; Beisi Zhou ***@***.***>; SDK Release Owners ***@***.***>; Jack Lichwa ***@***.***>
Subject: RE: MHSM Multi-Region HA Support

I’d like to leave it to Keyvault/MHSM service PM @Jack ***@***.***> @Nicholas ***@***.***> to decide whether other language SDKs are needed.
And here’s the link if you want to request release: https://portal.azure-devex-tools.com/tools/request-api-release<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fportal.azure-devex-tools.com%2Ftools%2Frequest-api-release&data=05%7C01%7Cyuchaoyan%40microsoft.com%7C47ff7760e9b248e6b75108db23717ee8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638142742604173056%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=eW7O3cczfnyFO1hJxiwk0OlngHIzyfnKK4FIbwrmHvE%3D&reserved=0>

@Josephine ***@***.***> @Yuchao ***@***.***> Can Python SDK catch 3/24 (3/17) release train since the PR has been merged?

Thanks,
Yishi

From: Josephine Qiu ***@***.******@***.***>>
Sent: Monday, March 13, 2023 11:10 AM
To: Yishi Wang ***@***.******@***.***>>; Ritesh Chitlangia ***@***.******@***.***>>; Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>; Beisi Zhou ***@***.******@***.***>>; SDK Release Owners ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

Azure SDK serves not only CLI release and generation.  To serve better Azure developer, Azure SDK requires cross language coverage on all supported SDK languages.  If you release only Python SDK on the new API version, it leads to a leak on other languages.  Could you create a release request covering other languages asap?  If 3/24 is not feasible, we should make up for .NET, Go, Java, JS in the next monthly train.   @SDK Release ***@***.***> for awareness.

Regards,
Josephine Qiu

From: Yishi Wang ***@***.******@***.***>>
Sent: Monday, March 13, 2023 10:45 AM
To: Josephine Qiu ***@***.******@***.***>>; Ritesh Chitlangia ***@***.******@***.***>>; Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>; Beisi Zhou ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

Hi Josephine,

I just raised the SDK release request for Python. That’s all for CLI development. Not sure if PSH needs .NET SDK released or not. cc @Beisi ***@***.***>

Thanks,
Yishi

From: Josephine Qiu ***@***.******@***.***>>
Sent: Monday, March 13, 2023 9:59 AM
To: Yishi Wang ***@***.******@***.***>>; Ritesh Chitlangia ***@***.******@***.***>>; Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

Hi Yishi, have you ever raised an SDK release request upon the new API version? Could you share me the link to the request?  Except Python, I wonder if you are also working on .NET SDK release on the same date?  What about other supported languages, Go, Java, JS?  We expect service team could well plan for the cross language coverage with monthly SDK release trains.

Regards,
Josephine Qiu

From: Yishi Wang ***@***.******@***.***>>
Sent: Friday, March 10, 2023 10:25 AM
To: Ritesh Chitlangia ***@***.******@***.***>>; Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>
Cc: Josephine Qiu ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

Hi @***@***.***>, @***@***.***>

I just learned from @Yuchao ***@***.***> that SDK release schedule has been changed to once a month, it’s not released on demand anymore.
And the next release date is 3/24 but CLI code complete date is 3/29. Time is getting so tight🥲

Can we get the PR merged today so that maybe @Yuchao ***@***.***> will help to deliver the SDK in advance on 3/17 (thanks to our personal friendship…)

Thanks,
Yishi

From: Ritesh Chitlangia ***@***.******@***.***>>
Sent: Friday, March 10, 2023 4:36 AM
To: Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Yishi Wang ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

***@***.*** ***@***.***>
Here’s the swagger PR that @Kyle ***@***.***> has posted. [Hub Generated] Publish private branch 'kim-soohwan-keyvault-mhsm-Microsoft.KeyVault-2023-02-01' by kim-soohwan · Pull Request #22625 · Azure/azure-rest-api-specs (github.com)<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F22625&data=05%7C01%7Cyuchaoyan%40microsoft.com%7C47ff7760e9b248e6b75108db23717ee8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638142742604173056%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Uj1ow4%2BwLqpw7H7283TndNl2smGWtt8E6Hi83yEIcWQ%3D&reserved=0>

@***@***.***>, latest change to add new property ‘isPrimary’ is in this commit<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F22625%2Fcommits%2Fdfd404bdff081b66fe848a45c07517b2828dcb65&data=05%7C01%7Cyuchaoyan%40microsoft.com%7C47ff7760e9b248e6b75108db23717ee8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638142742604173056%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=NtCjp4R9PicUJNGSvdFdkCR0qwJu4bHbKz9zKqfYyUo%3D&reserved=0>. Please take a look to see if this requires any CLI changes.

Thanks,
Ritesh
From: Jack Richins ***@***.******@***.***>>
Sent: Wednesday, March 8, 2023 6:01 PM
To: Nicholas Kondamudi ***@***.******@***.***>>; Yishi Wang ***@***.******@***.***>>; Ritesh Chitlangia ***@***.******@***.***>>
Subject: Re: MHSM Multi-Region HA Support

We discussed in the meeting, swagger should be ready next week. The change to include the primary region pool should not need any CLI changes.

Thanks,
Jack


From: Jack Richins ***@***.******@***.***>>
Date: Wednesday, March 8, 2023 at 5:59 PM
To: Nicholas Kondamudi ***@***.******@***.***>>, Yishi Wang ***@***.******@***.***>>, Ritesh Chitlangia ***@***.******@***.***>>
Subject: Re: MHSM Multi-Region HA Support
+ @Ritesh ***@***.***>

Thanks,
Jack


From: Jack Richins ***@***.******@***.***>>
Date: Wednesday, March 8, 2023 at 5:34 PM
To: Nicholas Kondamudi ***@***.******@***.***>>, Yishi Wang ***@***.******@***.***>>
Subject: MHSM Multi-Region HA Support
@Nicholas ***@***.***> are the multi-region HA REST API changes swagger ready? Yishi had trouble finding them.

Regarding the change to list the primary MHSM pools and it’s region when listing pools; is there anything more than Yishi needs to know?

Thanks in advance,
Jack Richins

**Comment by hari-bodicherla** [comment]
Thanks @Yishi ***@***.***> for letting me know.

There is no hurry to release .NET SDK release for PSH.

Thanks,
Beisi

From: Yuchao Yan ***@***.***>
Sent: Monday, March 13, 2023 12:57 PM
To: Yishi Wang ***@***.***>; Josephine Qiu ***@***.***>; Ritesh Chitlangia ***@***.***>; Jack Richins ***@***.***>; Nicholas Kondamudi ***@***.***>; Kyle Kim ***@***.***>; Beisi Zhou ***@***.***>; SDK Release Owners ***@***.***>; Jack Lichwa ***@***.***>; Azure/azure-cli ***@***.***>
Subject: RE: MHSM Multi-Region HA Support

Hi @Yishi ***@***.***>, @Ritesh ***@***.***>

Python will catch 3/24 (3/17) release train this time. But Mgmt SDK follow https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/761/Management-SDK-Release-Cycle<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdev.azure.com%2Fazure-sdk%2Finternal%2F_wiki%2Fwikis%2Finternal.wiki%2F761%2FManagement-SDK-Release-Cycle&data=05%7C01%7CBeisi.Zhou%40microsoft.com%7Ca8ee0de2007649c7d09108db237f698e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638142802370375835%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=OI8U8VLjtnsJrWruXlVyxhJHH%2Ff0ZgyIMfcdDAYFMpk%3D&reserved=0> to release monthly now. So in the future, please make release request earlier and give us more time  to arrange the release.

Add all members of @***@***.***> for awareness about new SDK release policy.

Thanks,
Yuchao
From: Yishi Wang ***@***.******@***.***>>
Sent: Monday, March 13, 2023 11:18 AM
To: Josephine Qiu ***@***.******@***.***>>; Ritesh Chitlangia ***@***.******@***.***>>; Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>; Beisi Zhou ***@***.******@***.***>>; SDK Release Owners ***@***.******@***.***>>; Jack Lichwa ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

I’d like to leave it to Keyvault/MHSM service PM @Jack ***@***.***> @Nicholas ***@***.***> to decide whether other language SDKs are needed.
And here’s the link if you want to request release: https://portal.azure-devex-tools.com/tools/request-api-release<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fportal.azure-devex-tools.com%2Ftools%2Frequest-api-release&data=05%7C01%7CBeisi.Zhou%40microsoft.com%7Ca8ee0de2007649c7d09108db237f698e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638142802370375835%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=3bu6YV2Gww1GDSYYrHqYPAr1A0jmyEjBhICMm0GGwgs%3D&reserved=0>

@Josephine ***@***.***> @Yuchao ***@***.***> Can Python SDK catch 3/24 (3/17) release train since the PR has been merged?

Thanks,
Yishi

From: Josephine Qiu ***@***.******@***.***>>
Sent: Monday, March 13, 2023 11:10 AM
To: Yishi Wang ***@***.******@***.***>>; Ritesh Chitlangia ***@***.******@***.***>>; Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>; Beisi Zhou ***@***.******@***.***>>; SDK Release Owners ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

Azure SDK serves not only CLI release and generation.  To serve better Azure developer, Azure SDK requires cross language coverage on all supported SDK languages.  If you release only Python SDK on the new API version, it leads to a leak on other languages.  Could you create a release request covering other languages asap?  If 3/24 is not feasible, we should make up for .NET, Go, Java, JS in the next monthly train.   @SDK Release ***@***.***> for awareness.

Regards,
Josephine Qiu

From: Yishi Wang ***@***.******@***.***>>
Sent: Monday, March 13, 2023 10:45 AM
To: Josephine Qiu ***@***.******@***.***>>; Ritesh Chitlangia ***@***.******@***.***>>; Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>; Beisi Zhou ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

Hi Josephine,

I just raised the SDK release request for Python. That’s all for CLI development. Not sure if PSH needs .NET SDK released or not. cc @Beisi ***@***.***>

Thanks,
Yishi

From: Josephine Qiu ***@***.******@***.***>>
Sent: Monday, March 13, 2023 9:59 AM
To: Yishi Wang ***@***.******@***.***>>; Ritesh Chitlangia ***@***.******@***.***>>; Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

Hi Yishi, have you ever raised an SDK release request upon the new API version? Could you share me the link to the request?  Except Python, I wonder if you are also working on .NET SDK release on the same date?  What about other supported languages, Go, Java, JS?  We expect service team could well plan for the cross language coverage with monthly SDK release trains.

Regards,
Josephine Qiu

From: Yishi Wang ***@***.******@***.***>>
Sent: Friday, March 10, 2023 10:25 AM
To: Ritesh Chitlangia ***@***.******@***.***>>; Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>; Yuchao Yan ***@***.******@***.***>>
Cc: Josephine Qiu ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

Hi @***@***.***>, @***@***.***>

I just learned from @Yuchao ***@***.***> that SDK release schedule has been changed to once a month, it’s not released on demand anymore.
And the next release date is 3/24 but CLI code complete date is 3/29. Time is getting so tight🥲

Can we get the PR merged today so that maybe @Yuchao ***@***.***> will help to deliver the SDK in advance on 3/17 (thanks to our personal friendship…)

Thanks,
Yishi

From: Ritesh Chitlangia ***@***.******@***.***>>
Sent: Friday, March 10, 2023 4:36 AM
To: Jack Richins ***@***.******@***.***>>; Nicholas Kondamudi ***@***.******@***.***>>; Yishi Wang ***@***.******@***.***>>; Kyle Kim ***@***.******@***.***>>
Subject: RE: MHSM Multi-Region HA Support

***@***.*** ***@***.***>
Here’s the swagger PR that @Kyle ***@***.***> has posted. [Hub Generated] Publish private branch 'kim-soohwan-keyvault-mhsm-Microsoft.KeyVault-2023-02-01' by kim-soohwan · Pull Request #22625 · Azure/azure-rest-api-specs (github.com)<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F22625&data=05%7C01%7CBeisi.Zhou%40microsoft.com%7Ca8ee0de2007649c7d09108db237f698e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638142802370375835%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=x7GoOcIXZHWTn8v8vKG8i3QkZairsCZMc3FExd%2F%2FASQ%3D&reserved=0>

@***@***.***>, latest change to add new property ‘isPrimary’ is in this commit<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F22625%2Fcommits%2Fdfd404bdff081b66fe848a45c07517b2828dcb65&data=05%7C01%7CBeisi.Zhou%40microsoft.com%7Ca8ee0de2007649c7d09108db237f698e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638142802370375835%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2FGlH%2FJvGI%2Fb42uPxe5OgBQq%2BtszIM07fi70ZO0VmZCQ%3D&reserved=0>. Please take a look to see if this requires any CLI changes.

Thanks,
Ritesh
From: Jack Richins ***@***.******@***.***>>
Sent: Wednesday, March 8, 2023 6:01 PM
To: Nicholas Kondamudi ***@***.******@***.***>>; Yishi Wang ***@***.******@***.***>>; Ritesh Chitlangia ***@***.******@***.***>>
Subject: Re: MHSM Multi-Region HA Support

We discussed in the meeting, swagger should be ready next week. The change to include the primary region pool should not need any CLI changes.

Thanks,
Jack


From: Jack Richins ***@***.******@***.***>>
Date: Wednesday, March 8, 2023 at 5:59 PM
To: Nicholas Kondamudi ***@***.******@***.***>>, Yishi Wang ***@***.******@***.***>>, Ritesh Chitlangia ***@***.******@***.***>>
Subject: Re: MHSM Multi-Region HA Support
+ @Ritesh ***@***.***>

Thanks,
Jack


From: Jack Richins ***@***.******@***.***>>
Date: Wednesday, March 8, 2023 at 5:34 PM
To: Nicholas Kondamudi ***@***.******@***.***>>, Yishi Wang ***@***.******@***.***>>
Subject: MHSM Multi-Region HA Support
@Nicholas ***@***.***> are the multi-region HA REST API changes swagger ready? Yishi had trouble finding them.

Regarding the change to list the primary MHSM pools and it’s region when listing pools; is there anything more than Yishi needs to know?

Thanks in advance,
Jack Richins
